change/@azure-msal-browser-42470d82-f43d-4f4c-92dc-684f7f44da44.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
This change file shouldn't be needed given the only changes are to markdown

Reply:
The check is failing for some reason, what could this be?